samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -28,6 +28,9 @@ public partial class DocumentTranslationClient
         /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
         public virtual HttpPipeline Pipeline => _pipeline;
 
+        /// <summary> The HTTP Uri. </summary>
+        public virtual string Uri => _endpoint;
```

Comment:
The return type of `Uri` should be `Uri` according to this guideline: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-primitives-uri

Reply:
@ShivangiReja FYI, for your investigation.